### ISSUE TITLE
What WCAG rules fails if Screen Reader focus is going on hidden text which is either incorrect or redundant when user is navigating in scan/browser mode

### ISSUE THREAD

**Issue by rahulpundir019** [w3c/wcag#4483]
I found a issue where Screen Reader focus is going on hidden text which is either incorrect or redundant when user is navigating in scan/browser mode.

When I am trying to map this issue in WCAG, I am not able to find any appropriate guideline.

Closest guidelines are Meaningful Sequence, Name role value and Info & Relationships but need second opinion?

**Comment by GreggVan** [comment]
I  think that it should be part of 1.3.1   and say that everything visible needs to be programmatically accessible, and everything that is not visible, should not be programmatically accessible, or should be marked clearly as non-visual at this time. But I really think it should be not programmatically accessible at all. Otherwise, a page that uses expanding paragraphs would be very usable by someone with vision, but unusable by someone who was using a screen reader because it would read everything not just the expanded topics which could mean a 200 or 300 page document with collapsing paragraphs would be one or two pages for someone who sees but two or 300 pages for someone using a screen reader. We don't say this explicitly, but we should either put it as a note or put it in understanding as this. And in. WCAG3 we need something specific.
